apps/docs/.vitepress/config.ts

```diff
@@ -421,6 +421,10 @@ export default defineConfig({
         link: '/guide/tooling/',
         collapsed: false,
         items: [
+          {
```

Comment:
I wanted to put this at the end of the array because it's arguably less important than testing and the CLI, but there is a [bug](https://github.com/FuelLabs/github-actions/issues/30) that prevented me from doing it.